<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiClassRegister</class>
 <widget class="QWidget" name="QtGuiClassRegister">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiClassRegister</string>
  </property>
  <widget class="QPushButton" name="registerAccountButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Register Account</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>81</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="user_label">
      <property name="text">
       <string>User</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="password_label">
      <property name="text">
       <string>Passord</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="firtName_Label">
      <property name="text">
       <string>FirstName</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lastName_label">
      <property name="text">
       <string>LastName</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="phoneNumber_label">
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="user_filed"/>
    </item>
    <item>
     <widget class="QLineEdit" name="password_field">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="firstName_field"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lastNameField"/>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneNumber_field"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
